mdbook-tera-backend/README.md

```diff
@@ -0,0 +1,82 @@
+# Tera backend extension for `mdbook`
+
+[![Visit crates.io](https://img.shields.io/crates/v/mdbook-i18n-helpers?style=flat-square)](https://crates.io/crates/mdbook-tera-backend)
+[![Build workflow](https://img.shields.io/github/actions/workflow/status/google/mdbook-i18n-helpers/test.yml?style=flat-square)](https://github.com/google/mdbook-i18n-helpers/actions/workflows/test.yml?query=branch%3Amain)
+[![GitHub contributors](https://img.shields.io/github/contributors/google/mdbook-i18n-helpers?style=flat-square)](https://github.com/google/mdbook-i18n-helpers/graphs/contributors)
+[![GitHub stars](https://img.shields.io/github/stars/google/mdbook-i18n-helpers?style=flat-square)](https://github.com/google/mdbook-i18n-helpers/stargazers)
+
+This `mdbook` backend makes it possible to use
+[tera](https://github.com/Keats/tera) templates and expand the capabilities of
+your books. It works on top of the default HTML backend.
+
+## Installation
+
+Run
+
+```shell
+$ cargo install mdbook-tera-backend
+```
+
+## Usage
+
+### Configuring the backend
+
+To enable the backend, simply add `[output.tera-backend]` to your `book.toml`,
+and configure the place where youre templates will live. For instance
+`theme/templates`:
+
+```toml
+[output.html] # You must still enable the html backend.
+[output.tera-backend]
+template_dir = "theme/templates"
+```
+
+### Creating templates
+
+Create your template files in the same directory as your book.
+
+```html
+<!-- ./theme/templates/language_list.html -->
+<ul>
+  {% for identifier, language_name in get_context(key="output.i18n.languages")
+  %}
```

Comment:
Is `get_context` still there? I think it got removed and that this can be replaced with

```suggestion
  {% for identifier, language_name in get_context(key="ctx.config.output.i18n.languages") %}
```

I don't think you show the corresponding `book.toml` for this anywhere? This makes the example a bit complicated in my opinion.

Reply:
We can fix this before/after merging this PR — we'll have time to go over the documentation before we release this on crates.io.